portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +10,10 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    private String first_name = "Tommy";
```

Comment:
This string could be final since it is a global variable and it is never changed.